app-leechcraft/liblaretz/liblaretz-0.1.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Use repoman to commit to fix this copyright header automatically. This is the old form.